docs/12.0/guides/upgrade-and-migration/migrating-from-11.1-to-12.0.md

```diff
@@ -220,3 +220,17 @@ To keep the previous (pre-12.0) behavior of a default keyboard shortcut, use the
 - [Remove a default keyboard shortcut](@/guides/accessories-and-menus/keyboard-shortcuts.md#removing-a-keyboard-shortcut)
 - [Replace a default keyboard shortcut](@/guides/accessories-and-menus/keyboard-shortcuts.md#replacing-a-keyboard-shortcut)
 - [Block a default keyboard shortcut's action](@/guides/accessories-and-menus/keyboard-shortcuts.md#blocking-a-keyboard-shortcut-s-actions)
+
+## Step 5: Don't refer to Walkontable
+
+Handsontable 12.0.0 makes it clear that Handsontable's rendering engine (internally referred to as "Walkontable") is not a part of Handsontable's public API.
+
+To emphasize this, we changed the following property name:
+
+| Before        | After          |
+| ------------- | -------------- |
+| `hot.view.wt` | `hot.view._wt` |
+
+#### Migrating to Handsontable 12.0
+
+Referring to Walkontable has never been a supported customization method. Use Handsontable's public APIs instead.
```

Comment:
This sentence sounds a bit too insensitive to the fact that up to the version 12.0.0, there was an example of using `hot.view.wt` in our docs. See the PR https://github.com/handsontable/handsontable/pull/9201 that contains changes to the docs, 4 editors, and most of the plugins due to this change. The migration guide should present some of the mitigations that a developer can do similarly to what we did in the PR 9201.

Reply:
@warpech got it, how about this? [5ea36f6](https://github.com/handsontable/handsontable/pull/9550/commits/5ea36f6d2e9e89715a610fd0a682378a0719d285)

For implementation examples (or other mitigations), I'd need more input from you or @budnix